test/unit/mdc-icon-button/foundation.test.js

```diff
@@ -22,6 +22,7 @@ import {verifyDefaultAdapter} from '../helpers/foundation';
 import MDCIconButtonToggleFoundation from '../../../packages/mdc-icon-button/foundation';
 
 const {strings} = MDCIconButtonToggleFoundation;
+const {cssClasses} = MDCIconButtonToggleFoundation;
```

Comment:
Change above line to `const {cssClasses, strings} = ...` and you don't need this line